modules/ROOT/pages/digger/proc_adding-a-jenkinsfile.adoc

```diff
@@ -0,0 +1,132 @@
+= Adding a Jenkinsfile
+
+To build a mobile app using the {digger-service} service, you must add a `Jenkinsfile` to your git repository, typically in the root directory of that repository.
+
+== Sample Android Jenkinsfile for Debug Build Type
```

Comment:
should we mention that both of these example `Jenkinsfile`s are based upon our showcase apps, and that their apps may be different in structure?

Reply:
added:
The following sample files are suitable for the link:showcase-apps.html[Showcase Apps]. You may need a different configuration for your mobile app.